src/Components/TwinPropertySearchDropdown/TwinPropertySearchDropdown.tsx

```diff
@@ -108,7 +109,7 @@ const TwinPropertySearchDropdown = (
     const searchTwinAdapterData = useAdapter({
         adapterMethod: (params: AdapterMethodParamsForSearchADTTwins) =>
             adapter.searchADTTwins(params),
-        refetchDependencies: [],
+        refetchDependencies: [searchPropertyName],
```

Comment:
Why would you want to re-fetch here? This method is done on click.

Reply:
this is in my twin property search. :)